Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -157,6 +193,13 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 	TickClientFPSCheck(DeltaSeconds);
 	TickPlayersConnectedCheck(DeltaSeconds);
 	TickUXMetricCheck(DeltaSeconds);
+	TickActorCountCheck(DeltaSeconds);
+
+	if (HasAuthority() &&
+		PrintMetricsTimer.HasTimerGoneOff())
```

Comment:
This logic seems a little confusing - we're checking if it's "gone off" - but then just immediately resetting it. Why isn't the print logic tied up here?

Reply:
Duplicate